content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -44,73 +45,73 @@ To fetch the logs, use the `kubectl logs` command, as follows:
 ```shell
 kubectl logs counter
 ```
+
 The output is:
-```
+
+```console
 0: Mon Jan  1 00:00:00 UTC 2001
 1: Mon Jan  1 00:00:01 UTC 2001
 2: Mon Jan  1 00:00:02 UTC 2001
 ...
 ```
 
-You can use `kubectl logs` to retrieve logs from a previous instantiation of a container with `--previous` flag, in case the container has crashed. If your pod has multiple containers, you should specify which container's logs you want to access by appending a container name to the command. See the [`kubectl logs` documentation](/docs/reference/generated/kubectl/kubectl-commands#logs) for more details.
+You can use `kubectl logs --previous` to retrieve logs from a previous instantiation of a container. If your pod has multiple containers, specify which container's logs you want to access by appending a container name to the command. See the [`kubectl logs` documentation](/docs/reference/generated/kubectl/kubectl-commands#logs) for more details.
 
 ## Logging at the node level
 
 ![Node level logging](/images/docs/user-guide/logging/logging-node-level.png)
 
-Everything a containerized application writes to `stdout` and `stderr` is handled and redirected somewhere by a container engine. For example, the Docker container engine redirects those two streams to [a logging driver](https://docs.docker.com/engine/admin/logging/overview), which is configured in Kubernetes to write to a file in json format.
+A container engine handles and redirects any output generated to a containerized application's `stdout` and `stderr` streams.
+For example, the Docker container engine redirects those two streams to [a logging driver](https://docs.docker.com/engine/admin/logging/overview), which is configured in Kubernetes to write to a file in JSON format.
```

Comment:
Aside: the written format is actually [JSONlines](https://jsonlines.org/) (the Docker documentation is also incorrect).
Each log entry is a JSON document, separated by a newline.

Reply:
OK. Is there more information about the use of JSON Lines in Kubernetes? I am happy to update if I can find more information.
What are your thoughts about removing the two task pages?
Is there a need to create new task pages?
I am looking at these links (no mention of JSON Lines):
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging#json-output-format
https://kubernetes.io/docs/concepts/cluster-administration/system-logs/
This pull request is mostly focused on cleanup.

Reply:
gotchu with an approve, tim!